depot.json

```diff
@@ -0,0 +1 @@
+{ "id": "58j0517pw2" }
```

Comment:
Having this file at the root means that running `depot build` anywhere inside this repo will default to that project ID - it's possible to provide this to the action directly if you'd prefer to skip the file.

Reply:
Good to know, I'll leave it in for now but I occasionally prune the root directory of stuff that isn't strictly necessary so that the `README.md` doesn't fall below the fold on GitHub.